packages/google_maps_flutter/ios/Classes/GoogleMapController.m

```diff
@@ -81,10 +82,17 @@ - (instancetype)initWithFrame:(CGRect)frame
     _markersController = [[FLTMarkersController alloc] init:_channel
                                                     mapView:_mapView
                                                   registrar:registrar];
+    _polylinesController = [[FLTPolylinesController alloc] init:_channel
+                                                        mapView:_mapView
+                                                      registrar:registrar];
     id markersToAdd = args[@"markersToAdd"];
     if ([markersToAdd isKindOfClass:[NSArray class]]) {
       [_markersController addMarkers:markersToAdd];
     }
+    id polylinesToAdd = args[@"polylinesToAdd"];
+    if ([polylinesToAdd isKindOfClass:[NSArray class]]) {
```

Comment:
Should we assert and throw if this check fails? Same questions to below from line 136 to line 149.